### PR TITLE
Throw exceptions on missing options

### PR DESCRIPTION
**Pull request by MilapNaik**
Fixes https://github.com/rokucommunity/roku-deploy/issues/149

### CODE REVIEW

File: src/RokuDeploy.ts [hunk 1]
```diff
@@ -237,9 +235,10 @@ export class RokuDeploy {
      * @param options
      */
     public async sideload(options: SideloadOptions): Promise<{ message: string; results: any }> {
+        this.checkRequiredOptions(options, ['host', 'password']);
         options = this.getOptions(options) as any;
         if (!options.host) {
-            throw new errors.MissingRequiredOptionError('must specify the host for the Roku device');
+            throw new Error('must specify the host for the Roku device');
```
Comment (TwitchBronBron) [inline src/RokuDeploy.ts:241]: This is handled by checkRequiredOptions, right? I see several other places in this file that do this check. I think those can all be removed in favor of checkRequiredOptions.

**Review by TwitchBronBron — CHANGES_REQUESTED** [review]
Just one suggestion.

File: src/RokuDeploy.ts (hunk 1 above)
Comment (MilapNaik) [reply src/RokuDeploy.ts:241]: Oh oops I meant to do this. Will remove

Comment (MilapNaik) [reply src/RokuDeploy.ts]: Fixed

**Review by TwitchBronBron — APPROVED** [review]

**Pull request merged**